stockflux-securities-master/src/components/InputForm.css

```diff
@@ -32,19 +32,28 @@
   font-weight: var(--font-weight-bolder);
   margin: var(--margin-small);
   flex-basis: 80px;
-  }
+}
 
+.input-form-input-read-only,
```

Comment:
I think the better approach here, which I think is more common, would be to have a base set of styling on the `.input-form-input` class which would dictate the style of the input in its default state and then you would *add* the `.input-form-input-read-only` class to it as a modifier which would provide some additional styling, overriding colours etc. In your HTML/JSX you should have the base class on all inputs and only add the extra class when you need to (in this case, when something is read only)

Reply:
yep done